addons/base_import_module/models/ir_module.py

```diff
@@ -445,7 +445,7 @@ def button_immediate_install_app(self):
 
     @api.model
     def _get_missing_dependencies(self, zip_data):
-        modules, unavailable_modules = self._get_missing_dependencies_modules(zip_data)
+        _modules, unavailable_modules = self._get_missing_dependencies_modules(zip_data)
```

Comment:
useless diff

Reply:
No, since `modules` is not used anymore, the test would fail